Cargo.lock

```diff
@@ -2731,14 +2799,25 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "644f9158b2f133fd50f5fb3242878846d9eb792e445c893805ff0e3824006e35"
 
 [[package]]
-name = "hkd32"
-version = "0.6.0"
+name = "hidapi"
```

Comment:
This dependency requires additional system package called libudev. On unix based machines, if you don't have that package the project will not compile. It's better to mention it in documentation. For dpkg based distributions you can find it as libudev-dev. For rpm based ones you can find it as libudev-devel.

Reply:
Thanks! This also reminded me that it's also required to explicitly install protoc now, which I also added to dev environment doc.